Airports/EGDO/Sector.txt

```diff
@@ -0,0 +1,7 @@
+CIRCLE_SECTORLINE:DOTWR:EGDO:2.5
+DISPLAY:DOTWR:DOTWR:DOAPP
+DISPLAY:DOAPP:DOTWR:DOAPP
+
+CIRCLE_SECTORLINE:DOAPP MATZ CIRCLE:EGDO:5
+DISPLAY:Swanwick Mil South West:Swanwick Mil South West:DOAPP
+DISPLAY:DOAPP:Swanwick Mil South West:DOAPP
```

Comment:
```suggestion
```

Same as for Merryfield - I would suggest you don't need this at all if EGDR _always_ covers top-down?